src/Proxy.hs

```diff
@@ -2062,17 +2077,7 @@ getAppSettingsWorker fucAndroid fucIOS = do
 
 getEpochLengthR :: Handler TypedContent
 getEpochLengthR =
-    runGRPC doGetEpochLength $ \(v :: Duration) -> do
-      let epochLengthObject = object ["epochLength" .= v]
+    runGRPC getConsensusInfo $ \cInfo -> do
+      let epochLengthObject = object ["epochLength" .= csEpochDuration cInfo]
       $(logInfo) "Successfully got epoch length."
```

Comment:
There are the following to choose from:

```
logSuccess :: MonadIO m => [String] -> m ()
logInfo :: MonadIO m => [String] -> m ()
logWarn :: MonadIO m => [String] -> m ()
logError :: MonadIO m => [String] -> m ()
logFatal :: MonadIO m => [String] -> m a
```

Should this be `logWarn` then?

Reply:
Well there are other levels which we use already
```
  $(logOther "Trace") $ "Invoking queries with headers: " <> Text.pack (show cookies)
```

This log should either be removed or moved to trace. I think it's only here because it was used for debugging the original implementation.